python/pip_install/extract_wheels/lib/utilities.py

```diff
@@ -0,0 +1,24 @@
+from argparse import ArgumentParser
```

Comment:
nit: instead of utilities.py can this be arguments.py since we hope to not grow other random utilities?

Reply:
😂  sure!